.travis.yml

```diff
@@ -72,11 +72,8 @@ script:
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -N deb/*.deb
     skip_cleanup: true
     on:
         tags: true
-        condition: >
-            "$TRAVIS_OS_NAME" = "linux" -a
-            "$CC" = "gcc"
-
+        condition: $TRAVIS_OS_NAME = 'linux' && $CC = 'gcc'
```

Comment:
Ah. So is it quoting of the conditions which maybe messes up the if statement? 
The parsing error for the `-a` syntax is : 

```
[0K/home/travis/.travis/job_stages: line 558: conditional binary operator expected
/home/travis/.travis/job_stages: line 558: expected `)'
/home/travis/.travis/job_stages: line 558: syntax error near `-a'
/home/travis/.travis/job_stages: line 558: `  if [[ ("$TRAVIS_OS_NAME = linux" -a "$CC = gcc"'
/home/travis/.travis/job_stages: line 558: conditional binary operator expected
/home/travis/.travis/job_stages: line 558: expected `)'
```